lib/deleteFile.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const axios = require('axios');
+
+const deleteFile = (delete_id, filetype, accountId, apiKey) => {
+  var type = '';
+  switch (filetype) {
+    case 'video':
+      type = 'stream';
+      break;
+    case 'image':
+      type = 'images/v1';
+      break;
+    default:
+      throw new Error('Unhandled File Type');
+  }
+  return axios({
+    method: 'DELETE',
+    url: `https://api.cloudflare.com/client/v4/accounts/${accountId}/${type}/${delete_id}`,
+    headers: {
+      Authorization: `Bearer ${apiKey}`,
+    },
+  }).catch((error) => {
+    if (error.status === 404) {
+      console.log(`Image not found on Cloudflare: ${error.message}`);
```

Comment:
I think we should still throw this error

I actually think we can probably remove the `catch` completely here and just throw error to the calling function

Reply:
removed this catch, errors will just throw when they occur and handling can be done within the caller